feat(changelog): kubernetes-deprecated-kubernetes-worker-nodes-with-nat- 2024-06-03 (#3258)
scaleway/docs-content: changelog/june2024/2024-06-03-kubernetes-deprecated-kubernetes-worker-nodes-with-nat-.mdx

---
title: Kubernetes worker nodes with NAT IPs
status: deprecated
author:
    fullname: 'Join the #k8s channel on Slack.'
    url: 'https://slack.scaleway.com'
date: 2024-06-03
category: containers
product: kubernetes
---

> Starting this deprecation cycle, existing Kubernetes Kapsule or Kosmos worker nodes can move from NAT IPs to Routed IPs. 

Reviewer: ```suggestion
Starting this deprecation cycle, existing Kubernetes Kapsule or Kosmos worker nodes can be moved from NAT IPs to Routed IPs. 
```

With Routed IPs, your worker nodes will receive a flexible IP address, maintaining IP consistency even if the Instance is restarted or the hypervisor is changed.

> NAT IPs will be at End of Life on October 1st, 2024. 

Reviewer: ```suggestion
NAT IPs will reach end of life on October 1st, 2024. 
```

> - All information on the migration methods provided in [our reference documentation](https://www.scaleway.com/en/docs/containers/kubernetes/reference-content/move-kubernetes-nodes-routed-ip/).

Reviewer: ```suggestion
- All information on the migration methods provided in our [documentation about moving Kubernetes nodes to routed IPs](/containers/kubernetes/reference-content/move-kubernetes-nodes-routed-ip/).
```

> - Find more about the changes behind this evolution [on our blog post](https://www.scaleway.com/en/news/routed-ips-are-coming-to-all-scaleway-products/)

Reviewer: ```suggestion
- [Read our blog post](https://www.scaleway.com/en/news/routed-ips-are-coming-to-all-scaleway-products/) to find out more about the changes behind this evolution.
```
